server.py

```diff
@@ -102,7 +102,7 @@ def ValidateListSurveyData(data):
     for x in data:
         if isinstance(x, dict):
             for k, v in x.items():
-                if not isinstance(k, str) or not isinstance(v, (str, list, int, float)):
+                if not isinstance(k, str) or (v and not isinstance(v, (str, list, int, float))):
```

Comment:
We shouldn't be getting null values from eQ, worth checking with @nealedj why we are.

Reply:
David has raised a card to investigate on the eQ side, in the mean time we could keep this or consider removing the validation - it's not providing much value if it can be None or pretty much any Python data type. I think we should raise a card in our backlog to create a schema that defines the structure we expect and use that for validation.

Reply:
Agreed will change so it just looks for a list of values